Sources/Authgear.swift

```diff
@@ -12,6 +13,7 @@ struct AuthorizeOptions {
     let prompt: String?
     let loginHint: String?
     let uiLocales: [String]?
+    let preferSFSafariViewController: Bool
```

Comment:
It is more conventional to add `s` after `prefer`, see https://developer.apple.com/documentation/uikit/uiview/1622449-clearscontextbeforedrawing for example.